Userland/Libraries/LibSQL/SQLResult.h

```diff
@@ -78,7 +79,7 @@ struct SQLError {
         default:
             VERIFY_NOT_REACHED();
         }
-        if (!error_argument.is_null() && error_argument == "") {
+        if (!error_argument.is_null() && error_argument != "") {
```

Comment:
Ah, I see this fixes a comment I made on an earlier commit. This should be squashed into that commit so that the typo never exists in the first place

Reply:
Yep